content/docs/05.developer-guide/06.storage.md

```diff
@@ -140,47 +140,65 @@ tasks:
-- The [Python](../../plugins/core/tasks/scripts/io.kestra.core.tasks.scripts.Python.md) task will run a Python script.
+Those script tasks are available in the [Scripts Plugin](https://github.com/kestra-io/plugin-scripts). Here is documentation for each of them:
+- The [Python](../../plugins/plugin-scripts-python/tasks/io.kestra.plugin.scripts.python.Script.md) task will run a Python script in a Docker container or in a local process.
+- The [Node](../../plugins/plugin-scripts-node/tasks/io.kestra.plugin.scripts.node.Script.md) task will run a Node.js script in a Docker container or in a local process.
+- The [Shell](../../plugins/plugin-scripts-shell/tasks/io.kestra.plugin.scripts.shell.Script.md) task will execute a single command, or a list of commands that you provide.
 
-The following example will query the BigQuery public dataset for Wikipedia pages, convert it to CSV, and use the file inside a Python task. The Python task will install the Pandas library and process the CSV file.
+The following example will query the BigQuery public dataset with Wikipedia page views to find the top 10 pages, convert it to CSV, and use the CSV file inside a Python task for further transformations using Pandas.
 
 ```yaml
 id: wikipedia-top-ten-python-panda
-namespace: io.kestra.tests
-description: A flow that loads wikipedia top 10 EN pages
+namespace: dev
+description: analyze top 10 Wikipedia pages
+
 tasks:
-  - id: query-top-ten
+  - id: query
     type: io.kestra.plugin.gcp.bigquery.Query
     sql: |
       SELECT DATETIME(datehour) as date, title, views FROM `bigquery-public-data.wikipedia.pageviews_2023`
       WHERE DATE(datehour) = current_date() and wiki = 'en'
       ORDER BY datehour desc, views desc
       LIMIT 10
     store: true
+    projectId: geller
+    serviceAccount: "{{envs.gcp_creds}}"
+
   - id: write-csv
     type: io.kestra.plugin.serdes.csv.CsvWriter
-    from: "{{outputs['query-top-ten'].uri}}"
-  - id: "python"
-    type: "io.kestra.core.tasks.scripts.Python"
-    inputFiles:
-      data.csv: "{{outputs['write-csv'].uri}}"
-      main.py: |
+    from: "{{outputs.query.uri}}"
+
+  - id: wdir
+    type: io.kestra.core.tasks.flows.WorkingDirectory
+    tasks:
+    - id: file
+      type: io.kestra.core.tasks.storages.LocalFiles
+      inputs:
+        data.csv: "{{outputs['write-csv'].uri}}"
+    
+    - id: pandas
+      type: io.kestra.plugin.scripts.python.Script
+      warningOnStdErr: false
+      docker: 
+        image: ghcr.io/kestra-io/pydata:latest
+      script: |
         import pandas as pd
         from kestra import Kestra
-        data = pd.read_csv("data.csv")
-        data.info()
-        sumOfViews = data['views'].sum()
-        Kestra.outputs({'sumOfViews': int(sumOfViews)})
-    requirements:
-      - pandas
+        
+        df = pd.read_csv("data.csv")
+        df.head(10)
+        views = df['views'].max()
```

Comment:
Max will show the most popular page. I found the sum weird as this would sum apples and oranges together into one metric (views for totally different pages). But done - it's sum now ✅